packages/network/connman/package.mk

```diff
@@ -68,7 +68,7 @@ PKG_CONFIGURE_OPTS_TARGET="WPASUPPLICANT=/usr/bin/wpa_supplicant \
 
 
 PKG_MAKE_OPTS_TARGET="storagedir=/storage/.cache/connman \
-                      statedir=/run/connman"
+                      statedir=/var/cache"
```

Comment:
@sraue / @seo - if you know why we were creating two `resolv.conf` files please advise.

With the current setup, the system creates `/var/cache/resolv.conf` (sym linked from `/etc/resolv.conf`) and populates it prior to starting `connman` with "fallback" nameservers `8.8.8.8` and `8.8.4.4`. `connman` itself is using and updating it's own `/run/connman/resolv.conf` file, and apparently ignoring `/var/cache/resolv.conf`.

With this change, there is now only one resolv.conf for the entire system: `/var/cache/resolv.conf`.

This file is initially populated with the same fallback nameservers as before, but once a valid network configuration is loaded (ie. manual or dhcp etc.) then these nameservers are overwritten by `connman`.

Sane?